index.html

```diff
@@ -5,7 +5,133 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="build/style.css" />
-    <title>Document</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Genshin Summit</title>
   </head>
-  <body></body>
+  <body>

```

Comment:
fixing the body width to 100vw doesn't solve the issue. **Have you tried it first?** Please, be careful not to mislead students who want to learn.

Reply:
Greetings @andrianarivo 😃, it is well-established in the programming community that there is rarely a singular definitive solution to a problem. It is important to note, however, that this does not imply that alternative approaches should be dismissed as erroneous because the individual proposing them may have more experience with the particular problem at hand or somethings in the former approach is not right. I would suggest that you exercise caution in selecting your phrasing to effectively convey your intended message. 🙂